[ERROR] 2018-12-27 08:51:59,277 [DubboClientReconnectTimer-thread-1]com.alibaba.dubbo.remoting.transport.AbstractClient. -->  [DUBBO] client reconnect to 192.168.100.1:20881 find error . url: dubbo://192.168.100.1:20881/com.shyl.msc.dm.service.IProductVendorService?accepts=500&anyhost=true&application=msc-provider&check=false&codec=dubbo&default.check=false&default.payload=11557050&dubbo=2.5.8&generic=false&heartbeat=60000&interface=com.shyl.msc.dm.service.IProductVendorService&iothreads=500&loadbalance=leastactive&logger=slf4j&methods=findByProduct,updateWithInclude,updateBatchWithInclude,merge,limit,saveOrUpdate,listByVendor,listByGPO,query,count,list,getById,listByVendorAndCode,mergeWithInclude,updateBatch,findByKey,save,update,delete,mergeWithExclude,queryByVendor,updateWithExclude,saveBatch,getByKey,findByStatus,findList,mapByVendor&pid=13788&register.ip=192.168.100.1&remote.timestamp=1545806794471&side=consumer&timeout=1200000&timestamp=1545806870262, dubbo version: 2.5.8, current host: 192.168.100.1 [] 
com.alibaba.dubbo.remoting.RemotingException: client(url: dubbo://192.168.100.1:20881/com.shyl.msc.dm.service.IProductVendorService?accepts=500&anyhost=true&application=msc-provider&check=false&codec=dubbo&default.check=false&default.payload=11557050&dubbo=2.5.8&generic=false&heartbeat=60000&interface=com.shyl.msc.dm.service.IProductVendorService&iothreads=500&loadbalance=leastactive&logger=slf4j&methods=findByProduct,updateWithInclude,updateBatchWithInclude,merge,limit,saveOrUpdate,listByVendor,listByGPO,query,count,list,getById,listByVendorAndCode,mergeWithInclude,updateBatch,findByKey,save,update,delete,mergeWithExclude,queryByVendor,updateWithExclude,saveBatch,getByKey,findByStatus,findList,mapByVendor&pid=13788&register.ip=192.168.100.1&remote.timestamp=1545806794471&side=consumer&timeout=1200000&timestamp=1545806870262) failed to connect to server /192.168.100.1:20881, error message is:No route to host: no further information
	at com.alibaba.dubbo.remoting.transport.netty.NettyClient.doConnect(NettyClient.java:124) ~[dubbo-2.5.8.jar:2.5.8]
	at com.alibaba.dubbo.remoting.transport.AbstractClient.connect(AbstractClient.java:268) ~[dubbo-2.5.8.jar:2.5.8]
	at com.alibaba.dubbo.remoting.transport.AbstractClient$1.run(AbstractClient.java:157) [dubbo-2.5.8.jar:2.5.8]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_111]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) [?:1.8.0_111]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_111]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) [?:1.8.0_111]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [?:1.8.0_111]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [?:1.8.0_111]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_111]
Caused by: java.net.NoRouteToHostException: No route to host: no further information
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method) ~[?:1.8.0_111]
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717) ~[?:1.8.0_111]
	at org.jboss.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.connect(NioClientSocketPipelineSink.java:384) ~[netty-3.2.5.Final.jar:?]
	at org.jboss.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processSelectedKeys(NioClientSocketPipelineSink.java:354) ~[netty-3.2.5.Final.jar:?]
	at org.jboss.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:276) ~[netty-3.2.5.Final.jar:?]
	... 3 more
[INFO ] 2018-12-27 08:51:59,951 [DubboZkclientConnector-SendThread()]org.apache.zookeeper.ClientCnxn. --> Unable to read additional data from server sessionid 0x167e82a1ea70023, likely server has closed socket, closing socket connection and attempting reconnect [] 
[INFO ] 2018-12-27 08:51:59,956 [DubboZkclientConnector-SendThread()]org.apache.zookeeper.ClientCnxn. --> Unable to read additional data from server sessionid 0x167e82a1ea70022, likely server has closed socket, closing socket connection and attempting reconnect [] 
